src/main/resources/application.yaml

```
@@ -45,6 +46,11 @@ spring:
 azure:
   application-insights:
     instrumentation-key: ${hmc.AppInsightsInstrumentationKey:00000000-0000-0000-0000-000000000000}
+  jms:
+    servicebus:
+      connection-string: ${HMCServiceBusConnectionString}
+      wait-to-retry-time: ${SERVICE_BUS_IDLE_TIMEOUT:60}
+      queue-name: ${HMCServiceBusQueueName}
```

```suggestion
      connection-string: ${HMC_SERVICE_BUS_CONNECTION_STRING}
      wait-to-retry-time: ${SERVICE_BUS_IDLE_TIMEOUT:60}
      queue-name: ${HMC_SERVICE_BUS_QUEUE}
```
For consistency with env var naming convention (need to update any references too)